### PR TITLE
Replace status button with madmin button

### PR DESCRIPTION
**Pull request by daangroot**
## Description

This PR replaces the currently broken status page button with a MADmin button. The MADmin url is configurable via a command line argument or the config file.

## How Has This Been Tested?

Tested on ubuntu server. Works :)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: static/js/map.js [hunk 1]
```diff
@@ -320,6 +320,12 @@ function initMap() { // eslint-disable-line no-unused-vars
         }, 500)
     })
 
+    if (showConfig.madmin) {
+        $('#madmin').each(function() {
```
Comment (na-ji) [inline static/js/map.js]: Why not put the URL directly on the template?

Comment (daangroot) [reply static/js/map.js]: Because the URL is not fixed.

**Comment by daangroot** [comment]
@na-ji I forgot about the `custom.js` file. Maybe it makes more sense to put the URL in there?

File: static/js/map.js (hunk 1 above)
Comment (na-ji) [reply static/js/map.js]: What do you mean it is not fixed?
What I see is that upon changing the URL, you still need to restart the app to make the change. So you could put the URL directly in the template, preventing the need to add more javascript to it.

**Comment by na-ji** [comment]
This file is not saved on the repository and can be changed by the users. I
do not think it's a good idea to ask the users to add more code.

File: static/js/map.js (hunk 1 above)
Comment (daangroot) [reply static/js/map.js]: Right, it's in the template now.

**Pull request merged**